src/main/java/tdl/datapoint/infra_events/processing/VideoProcessingFailedECSEvent.java

```diff
@@ -0,0 +1,49 @@
+package tdl.datapoint.infra_events.processing;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class VideoProcessingFailedECSEvent extends FailedECSEvent {
```

Comment:
I am not clear about the purpose of this class, why is this used in both test code and implementation code?
It would be good to have a small helper class in the test class to build the json event and a separate one implementation to read a json event, but they should not be combined.

Reply:
The implementation classes inherit from a common class, as they share the same implementation except for `challengeId` and `roundId`.